gcloud/logging/_gax.py

```diff
@@ -73,6 +73,8 @@ def list_entries(self, projects, filter_='', order_by='',
                   if not None, indicates that more entries can be retrieved
                   with another call (pass that value as ``page_token``).
         """
+        if page_token is None:
+            page_token = INITIAL_PAGE
```

Comment:
OK. I'm still going to keep digging because the error is very strange. I'll cc you on my fake PRs and/or builds to inspect further.